packages/swc-angular/src/index.ts

```diff
@@ -88,7 +88,7 @@ function assertCompatibleSwcCoreVersion(version: string) {
     }>(packageJsonPath).version;
   }
 
-  if (!(version.startsWith('1.4.') || version.startsWith('1.5.'))) {
+  if (!(version.startsWith('1.4.') || version.startsWith('1.5.') || version.startsWith('1.6.'))) {
     console.error(`
     @swc/core version ${version} is incompatible with @jscutlery/swc-angular.
     Please use @swc/core version 1.4.x or 1.5.x.
```

Comment:
We should update the log as well: 
```suggestion
    Please use @swc/core version 1.4.x or 1.5.x. or 1.6.x.
```

Reply:
Ok I will apply suggestion soon.

Reply:
I just did it I was on your branch, thanks for the work @pumano!